stack/tasks/create-a-m-o.js

```diff
@@ -74,7 +74,7 @@ const writeIndexJs = (path, _name) => {
 
         // Specify observed attributes so that attributeChangedCallback will work,
         // this is essential for external re-rendering trigger.
-        static get observedAttributes() { return []; }
+        static get observedAttributes() { return ['classes']; }
```

Comment:
@LucaMele 
do we really want custom classes in general?

Reply:
yep